.github/workflows/django-overhead-profile.yml

```diff
@@ -0,0 +1,86 @@
+name: django-overhead-profile
+on:
+  push:
+    branches: [master]
+  pull_request:
+jobs:
+  django-overhead-profile:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: trace-examples/python/django/django-simple
+    env:
+      DJANGO_ALLOWED_HOSTS: 127.0.0.1
+      DJANGO_SECRET_KEY: vNIF0IeM1IN9YAOOB8yEmNddSvnnXxx7E7D3TwWsdno
+      DATABASE_URL: sqlite:///django.db
+      DJANGO_SETTINGS_MODULE: config.settings.production
+      USE_DOCKER: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+
+      - uses: actions/checkout@v2
+        with:
+          repository: DataDog/trace-examples
+          ref: 760deae1fd2f2371cf813d3ff3ca9f0e040e8c60
+          path: trace-examples
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Prepare Django app
+        run: |
+          pip install -r requirements/production.txt
+          python manage.py migrate
+
+      - name: Install ddtrace
+        run: pip install -e ../../../../ddtrace
+
+      - name: Checkout Austin development branch
+        uses: actions/checkout@v2
+        with:
+          repository: P403n1x87/austin
+          ref: b029cb7728783cc1035a032c9689f272e6ca2de6
+          path: austin
+
+      - name: Compile Austin on Linux
+        run: |
+          cd $GITHUB_WORKSPACE/austin
+          gcc -Wall -O3 -Os -s -pthread src/*.c -o src/austin
+          cd -
+
+      - name: Install diff tool dependencies
+        run: |
+          pip install rich
+          pip install git+git://github.com/p403n1x87/austin-python.git@acf3f3d3263a583c30f5c95e4b3ad337a6793aa3
+
+      - name: Collect profiles
+        run: |
+          mkdir ${{ github.workspace }}/artifacts
+
+          gunicorn config.wsgi --pid gunicorn.pid &
+          sleep 1
+          sudo $GITHUB_WORKSPACE/austin/src/austin -i 200 -o ${{ github.workspace }}/artifacts/baseline.austin -sp $! -x 120
```

Comment:
I think it would be best to start a load tester at this point, before attaching Austin to gunicorn.

Also, sampling at 200 microseconds for about 2 minutes might generate quite a lot of data; I'd go for about 10 seconds.